app.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+require 'sinatra/base'
+require_relative './lib/peep.rb'
+require_relative './lib/user.rb'
+
+class ChitterApp < Sinatra::Base
+  get '/' do
+	 erb :index
+	end
+
+  get '/peeps' do
+    @feed = Peep.all 
+    erb :peeps
+  end
+
```

Comment:
Your app file is looking great, your controllers are skinny and you seem to have followed MVC to the letter